src/lib/network/edit_nic_name.rb

```diff
@@ -63,18 +63,11 @@ def run
           next
         end
 
-        if UI.QueryWidget(:udev_type, :CurrentButton) == :mac
-          rule_key = MAC_UDEV_ATTR
-          rule_value = @mac
-        else
-          rule_key = BUSID_UDEV_ATTR
-          rule_value = @bus_id
-        end
+        udev_type = UI.QueryWidget(:udev_type, :CurrentButton)
 
-        # update udev rules and other config
```

Comment:
Removed by error :-1:  will add it back after the code review with the changes that could be needed.